src/gui/qt/main_window.cpp

```diff
@@ -326,6 +328,7 @@ void MainWindow::initialise() {
 }
 
 void MainWindow::applyTheme() {
+#if QT_VERSION >= QT_VERSION_CHECK(6, 5, 0)
```

Comment:
I think this `#if` should be moved to just before line 346 (the comment starting `Don't load default-dark`)